msktutil.cpp

```diff
@@ -770,6 +803,8 @@ int main(int argc, char *argv [])
             exec->set_mode(MODE_CLEANUP);
         } else if (!strcmp(argv[1], "delete")) {
             exec->set_mode(MODE_DELETE);
+        } else if (!strcmp(argv[1], "reset")) {
+            exec->set_mode(MODE_RESET);
```

Comment:
Do we really need a new mode for `reset` rather than introducing a new option `--default-password` that can be used with `create` and `update` similar to `--password`?

Reply:
Given that no-one's come up with a better implementation, let's stick with the new command.